Changelogs for Digital Twin Manager UI 1020.3.4 (#3904)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020-2-3-updated-the-public-options-url-to-reference-the-new-endpoint.md

---
date: ""

> title: "Saved custom translations are now listed in Localization page"

Reviewer: ```suggestion
title: "Custom translations are now listed in the Localization page"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2541"
version: "1020.2.3"
---

> Previously, custom translations were not listed in the **Localization** page, so that users could not modify or delete them. This issue is now resolved, and the translations are properly shown.

Reviewer 2: With this changelog, I'm not able to understand what the actual issue was or what has been fixed. The line "Previously, custom translations were not listed in the Localization page" makes it sound like the feature never worked from the beginning, which implies the app wasn’t tested before release — and that’s not the impression we want to give customers.

A changelog should clearly describe the user-visible problem and then explain what was fixed. Clarifying the real behavior (e.g., translations disappearing after saving) would help customers and internal teams better understand the issue and the value of the fix.

Reviewer 3: @rahulc8y, this bug got introduced with the change in public-options app behavior made by cumulocity-ui. When a user adds translation strings, the translation continue to work but the keys were not visible in the localization page and users could not modify or delete them. Should I mention this to make it clear to the users?

Reviewer 3: ```suggestion
After the introduction of improved branding feature, custom translations in the **Localization** page of Digital Twin Manager application were not listed, and users could not modify or delete them. This issue is now resolved, and the translations are properly shown.
```

> Previously, custom translations in the Localization page were not being listed, preventing users from modifying or deleting the saved custom translations. This is now resolved and the translations are listed.

Reviewer: ```suggestion
Previously, custom translations were not listed in the **Localization** page, so that users could not modify or delete them. This issue is now resolved, and the translations are properly shown.
```

---
date: ""

> title: "Updated the public-options URL to reference the new endpoint."

Reviewer 3: ```suggestion
title: "Saved custom translations are now listed in Localization page"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2541"
version: "1020.2.3"
---
Updated the public-options URL for the translation directory service to

> point to the new endpoint

Reviewer 3: ```suggestion
Previously, custom translations in the Localization page were not being listed, preventing users from modifying or deleting the saved custom translations. This is now resolved and the translations are listed.
```
